kratos/containers/variable.h

```diff
@@ -258,7 +270,27 @@ class Variable : public VariableData
     ///@{
 
     /**
-     * This method returns the zero value of the variable type
+     * @brief This method returns the time derivative variable
+     * @return The pointer of the time derivative variable (if any)
+     */
+    const VariableType& GetTimeDerivative() const
+    {
+        KRATOS_DEBUG_ERROR_IF(mpTimeDerivativeVariable == nullptr) << "Time derivative for Variable \"" << Name() << "\" was not assigned" << std::endl;
+        return *mpTimeDerivativeVariable;
+    }
+
+    /**
+     * @brief This method returns the second time derivative variable
+     * @return The pointer of the second time derivative variable (if any)
+     */
+    const VariableType& GetSecondTimeDerivative() const
```

Comment:
we intended `GetTimeDerivative` to be called multiple times

I don't have a strong opinion abt this method

@KratosMultiphysics/technical-committee opinions?

Reply:
Initially I was in favour of having the second derivative too.

However, as it is not used as often as the first one, maybe we can live without the `GetSecondTimeDerivative` method (at least in a very first design). Although it is quite verbose to recursively get the second derivative, it would make things easier.